packager/signer/signer.go

```diff
@@ -477,11 +477,17 @@ func (this *Signer) serveSignedExchange(resp http.ResponseWriter, fetchResp *htt
 	if err != nil {
 		util.NewHTTPError(http.StatusInternalServerError, "Error building validity href: ", err).LogAndRespond(resp)
 	}
+	// Expires - Date must be <= 604800 seconds, per
+	// https://tools.ietf.org/html/draft-yasskin-httpbis-origin-signed-exchanges-impl-00#section-3.5.
+	duration := 7*24*time.Hour
```

Comment:
suggest spaces around * for readability.

Reply:
Okay, ran go fmt on these two files, which fixed this.